src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -385,112 +405,47 @@ class EntityClient (requestContext: RequestContext, modelSchema: ModelSchema, go
       op(pfbUrl)
     }
 
-    // probe Rawls with an empty upsert to this workspace. This checks that the workspace exists and that the user has
-    // permissions to perform an upsert.
-    def validateUpsertPermissions()(op: Boolean => PerRequestMessage) = {
-      val pipeline: WithTransformerConcatenation[HttpRequest, Future[HttpResponse]] = authHeaders(requestContext) ~> sendReceive
-
-      val probeUpsert = pipeline {
-        Post(FireCloudDirectiveUtils.encodeUri(Rawls.entityPathFromWorkspace(workspaceNamespace, workspaceName) + "/batchUpsert"), List.empty[String])
-      }
-
-      probeUpsert.map {
-        case resp if resp.status == NoContent => op(true)
-        case resp => RequestComplete(resp) // bubble up errors
-      }
-    }
-
     // validate presigned URL
     validatePfbUrl(pfbRequest) { pfbUrl =>
-      // empty-array batchUpsert to Rawls to verify workspace existence and permissions
-      validateUpsertPermissions() { _ =>
-        // generate job UUID
-        val jobId = java.util.UUID.randomUUID()
-
-        // the payload to Arrow needs to include the PFB url, job id, workspace info, and user info
-        val user = WorkbenchUserInfo(userInfo.id, userInfo.userEmail)
-        val arrowPayload = pfbRequest.copy(jobId = Option(jobId.toString),
-          workspace = Option(WorkspaceName(workspaceNamespace, workspaceName)),
-          user = Option(user))
-
-        // fire-and-forget call Arrow with UUID, workspace info, and presigned URL
-        val idToken = googleServicesDAO.getAdminIdentityToken
-        val gzipPipeline = addHeader (`Accept-Encoding`(gzip)) ~> addCredentials(OAuth2BearerToken(idToken)) ~> sendReceive ~> decode(Gzip)
-        gzipPipeline { Post(avroToRawlsURL, arrowPayload) }
-
-        // the response payload to the end user omits the userid/email
-        val responsePayload = arrowPayload.copy(user = None)
-
-        // return Accepted with UUID
-        RequestComplete(Accepted, responsePayload)
+      // verify workspace existence and permissions
+      validateUpsertPermissions(userInfo, workspaceNamespace, workspaceName, "write") { _ =>
+        // the payload to Import Service sends the path
+        val importServicePayload: ImportServiceRequest = ImportServiceRequest(path = pfbUrl.toString, filetype = "pfb")
+        // TODO: AS-155: always encode namespace/name
+        val importServiceUrl = s"${FireCloudConfig.ImportService.server}/$workspaceNamespace/$workspaceName/imports"
+
+        userAuthedRequest(Post(importServiceUrl, importServicePayload))(userInfo) map {
+          case resp if resp.status == Created =>
+            val importServiceResponse = unmarshal[ImportServiceResponse].apply(resp)
+            // for backwards compatibility, we return Accepted(202), not Created(201)
```

Comment:
also because it's still correct :)

Reply:
yeah - I guess I'm distinguishing here that import service returns 201, but orch returns  202